src/itensor/diagitensor.jl

```diff
@@ -84,12 +65,12 @@ In the case when `x isa Union{Int, Complex{Int}}`, by default it will
 be converted to `float(x)`. Note that this behavior is subject to change
 in the future.
 """
-function diagITensor(as::AliasStyle, eltype::Type{<:Number}, x::Number, is::Indices)
-  return diagITensor(AllowAlias(), eltype, fill(eltype(x), mindim(is)), is...)
+function diagITensor(::AliasStyle, ElT::Type{<:Number}, x::Number, is::Indices)
```

Comment:
I actually prefer the style of using `eltype` instead of `ElT`, when it is the argument of a function.

Reply:
Is the name `eltype` a typical variable name for this type of thing in julia? My reason for changing the variable name is that eltype is also a Base function name.

Reply:
I use it when I know I won't use the function `eltype` in the function, or if you want to use the function you can use `Base.eltype` explicitly. But more importantly I prefer the style of using lower case for argument names. So a compromise could be `elt` or something like that.